perf_bench_test.go

```diff
@@ -181,6 +181,7 @@ func TestColocateVMsOnSameCPU(t *testing.T) {
 	require.NoError(t, err, "Cannot get CPU info")
 	sibling, err := cpuInfo.GetSibling(*profileCPUID)
 	require.NoError(t, err, "Cannot run taskset")
+	require.NotEqual(t, sibling, -1, "SMT is off. Cannot allocate two VMs on a single core")
```

Comment:
the tool should support both SMT on and off.

Reply:
Done